src/cow-react/modules/application/containers/App/index.tsx

```diff
@@ -42,3 +44,10 @@ export function App() {
     </ErrorBoundary>
   )
 }
+
+export const App = withLDProvider({
+  clientSideID: LAUNCH_DARKLY_CLIENT_KEY,
+  options: {
+    bootstrap: 'localStorage',
+  },
+})(WrappedApp)
```

Comment:
Their provider wraps our whole app.
See their docs for reference https://docs.launchdarkly.com/sdk/client-side/react/react-web#initializing-using-withldprovider